pkg/util/version/const.go

```diff
@@ -25,14 +25,26 @@ const (
 
 var GitCommit = "unknown"
 
-// DefaultInstallStream describes stream we are defaulting to for all new clusters
-var DefaultInstallStream = &Stream{
-	Version:  NewVersion(4, 10, 63),
-	PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:340091aefa0bba06bbb99cc58cb1f2b73404c832f72b83c526b8e7677efbecef",
+// DefaultMinorVersion describes the minor OpenShift version to default to
+var DefaultMinorVersion = 11
+
+// DefaultInstallStreams describes the latest version of our supported streams
+var DefaultInstallStreams = map[int]*Stream{
+	10: {
+		Version:  NewVersion(4, 10, 63),
+		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:340091aefa0bba06bbb99cc58cb1f2b73404c832f72b83c526b8e7677efbecef",
+	},
+	11: {
+		Version:  NewVersion(4, 11, 44),
+		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:52cbfbbeb9cc03b49c2788ac7333e63d3dae14673e01a9d8e59270f3a8390ed3",
+	},
```

Comment:
I know this is already merged, but why add the 4.10 stuff to a `DefaultInstallStreams` struct if we're not setting 4.10 as a default anymore?  

We can't pass in "4.10" as an install version and it picks the version above.  It might confuse team member IMO.

Reply:
@bennerv I've done it like this because the discoverycache right now is 4.10 and so I want what the default install stream for 4.10 used to be. Maybe it doesn't make as much sense because you can't ask for "4.10" and get anything. (should we have that, maybe?)